cloudbuild.yaml

```diff
@@ -12,3 +12,34 @@ steps:
         fi
     env:
       - "TAG_NAME=${TAG_NAME}"
+
+  - id: deploy-csharp
+    name: "bitnami/dotnet-sdk"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        if [ "$TAG_NAME"  ] ; then
+          cp vmlapis.csproj gen/csharp
+          cd gen/csharp
+          dotnet pack -o . -c Release
+          dotnet nuget push vmlapis.$TAG_NAME.nupkg --source https://api.nuget.org/v3/index.json --api-key $$NUGET_API_KEY
+        fi
+    secretEnv: ['NUGET_API_KEY']
+
+  - id: deploy-java
+    name: "maven:3.6.3-jdk-8"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        if [ "$TAG_NAME" ] ; then
+          cp -R gen/java java-package/src/main/java
+          cd java-package
+          mvn deploy
+        fi
+
+availableSecrets:
+  secretManager:
+    - versionName: projects/161107618804/secrets/nuget-api-key/versions/latest
```

Comment:
I assume this is terraformed

Reply:
Indeed, this had to be created manually if I remember correctly just because it's an existing token from another system, but we can (and should) import it to terraform.

Reply:
wouldn't it mean it'll be plaintext in TF git?

Reply:
Yes, if we would create it like that. However, since it has been created via the console, I believe we can import it to terraform just as an object, without the actual value - but we'd have to investigate 😄